test/train-sets/ref/help.stdout

```diff
@@ -200,6 +200,12 @@ Audit Regressor:
                         this mode.
 Autolink:
   --autolink arg        create link function with polynomial d
+Debug: automl reduction:
+  --automl arg (=3, )                set number of live configs
+  --global_lease arg (=10, )         set initial lease for automl interactions
+  --cm_type arg (=interaction, )     set type of config manager
+  --priority_type arg (=none, )      set type of config manager in {none, le}
```

Comment:
whats `le`?

Reply:
also both arguments `cm_type` and `priority_type` have a description of set type of config manager, is that by mistake?

Reply:
`le` (least-exclusion) is a different priority algorithm to decide which config is swapped in next when one runs out of lease.

Reply:
And yes that's a type with the descriptions

Reply:
can we add the long name for `le` in the help?

Reply:
Sure